src/rootfs/files/scripts/log.sh

```
@@ -0,0 +1,48 @@
+
+function show_usage() {
+    echo "Running this script manually is wrong, please source it";
+    echo "Example:";
+    printf "\tsource $0\n";
+}
+
+function _log() {
+    local TYPE="$1";
+    local MSG="$2";
+
+    local DATE="$(date +'%Y-%m-%d %H:%M:%S')";
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)